changelog.txt

```diff
@@ -0,0 +1,5 @@
+important - Add ability to deploy functions individually or by export groups (e.g. firebase deploy --only functions:functionName). Run "firebase deploy --help" for details.

```

Comment:
I think this is just `feature` -- typically important would only be used to signify e.g. big breaking change or other "YOU MUST TAKE ACTION" material.